tests/countries/test_finland.py

```diff
@@ -165,6 +167,152 @@ def test_all_saints_day(self):
             "1957-11-01",
         )
 
+    def _test_unofficial_holiday(self, name, month_and_day, year_of_adoption):
```

Comment:
Could you refactor it to accept `name` and `since` params where since would have "yyyy-mm-dd" format. Basically I suggest merging str/int month_and_day/year_of_adoption to a single param for better readability.

Reply:
Ooh, so the input would be simply the date when this holiday was established. I like this, will refactor 👍